airflow/configuration.py

```diff
@@ -263,6 +265,12 @@ def run_command(command):
 # Filter the list of dags by owner name (requires authentication to be enabled)
 filter_by_owner = False
 
+# SSL certificate file
+ssl_certfile =
```

Comment:
Should these default to commented out, rather than empty?

    # ssl_certfile =

Seems to be the pattern with:

    # s3_log_folder =

Above.